### PR TITLE
problem solved for not going to poketop

### PR DESCRIPTION
**Pull request by anonymouslulz22**
Short Description: 
fix for not going to the pokestop
Fixes:
- 
- 
- 

so today i pulled the latest dev and tried to run and saw that the bot is just finding a fort and not moving to it! here is a fix for it and i have tested it and its working again

### CODE REVIEW

**Comment by gpires** [comment]
line 44 is not indented.

**Comment by anonymouslulz22** [comment]
yea! let me check out

**Comment by tstumm** [comment]
Your indention is wrong

**Comment by gpires** [comment]
```
        step_walker = self._step_walker
        if step_walker == True:
            self.api.player_update(latitude=lat, longitude=lng)
            response_dict = self.api.call()
            logger.log('Arrived at Pokestop')
            return WorkerResult.SUCCESS
        elif step_walker == False:
            return WorkerResult.RUNNING
```

Like this?

File: pokemongo_bot/cell_workers/move_to_fort_worker.py [hunk 1]
```diff
@@ -41,7 +41,13 @@ def work(self):
                 )
 
 
-            if not self._step_walker.step():
+        step_walker = self._step_walker
+            if step_walker == True:
+                self.api.player_update(latitude=lat, longitude=lng)
```
Comment (douglascamata) [inline pokemongo_bot/cell_workers/move_to_fort_worker.py]: please just add this `self.api.player_update` calls  after each step is taken by the `StepWalker` and revert all the other changes in this file.

File: pokemongo_bot/cell_workers/move_to_fort_worker.py [hunk 2]
```diff
@@ -41,8 +41,16 @@ def work(self):
                 )
 
 
-            if not self._step_walker.step():
+            step_walker = self._step_walker
+                if step_walker == True:
+                self.api.player_update(latitude=lat, longitude=lng)
```
Comment (douglascamata) [inline pokemongo_bot/cell_workers/move_to_fort_worker.py]: don't know if you saw my comment, but please move the `self.api.player_update` call to after the `StepWalker` finishes moving and leave this worker untouched.

**Comment by tstumm** [comment]
This does not fix the problem. We just found the problematic code and fixed it. PR is coming.

**Comment by anonymouslulz22** [comment]
ok!

**Pull request closed without merging**